### PR TITLE
MAT-8943: remove featureflag

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-8943](https://jira.cms.gov/browse/MAT-8943)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public Page<MeasureListDTO> searchMeasuresByCriteria(
 
     boolean nestedFlag =
         invocationSource.equals("testCase")
-            ? appConfigService.isFlagEnabled(MadieFeatureFlag.EDIT_TESTS_ON_VERSIONED_MEASURES)
+            ? true // Always allow nested search for test case copy page
             : appConfigService.isFlagEnabled(MadieFeatureFlag.MEASURE_SEARCH);
```
Comment (jkotanchik-SB) [inline src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java]:
```suggestion
boolean nestedFlag =
      invocationSource.equals("testCase") || appConfigService.isFlagEnabled(MadieFeatureFlag.MEASURE_SEARCH);
```

**Review by jkotanchik-SB — DISMISSED** [review]
Simplify nestedFlag boolean.

File: src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java [hunk 2]
```diff
@@ -61,18 +61,14 @@ public Page<MeasureListDTO> searchMeasuresByCriteria(
     LookupOperation lookupOperation = getLookupOperation();
     UnwindOperation unwindOperation = unwind("measureSet");
 
-    // Project only needed fields from Measure to improve performance
-    ProjectionOperation initialProjection = project().andExclude("testCases", "elmJson");
-    aggregationOperations.add(lookupOperation);
-    aggregationOperations.add(unwindOperation);
-    aggregationOperations.add(initialProjection);
-
```
Comment (etan-sb) [inline src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java]: I think we need this code (to exclude test cases and elmJson from search pipeline) - https://github.com/MeasureAuthoringTool/measure-service/pull/955. @RohitKandimalla , can you please confirm

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -73,7 +73,7 @@ management:
 
 okta:
   oauth2:
-    issuer: ${OKTA_ISSUER:https://integrator-3514157.okta.com/oauth2/default}
+    issuer: ${OKTA_ISSUER:https://dev-18092578.okta.com/oauth2/default}
```
Comment (etan-sb) [inline src/main/resources/application.yml]: Should we be changing the issuer here?

File: src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java (hunk 2 above)
Comment (RohitKandimalla) [reply src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java]: Yes we need to exclude testCases and elmJson.

**Review by etan-sb — APPROVED** [review]
